nss-copper-dragonflies/BangazonSite
# Seed Data

# Description

Added seed data

Fixes # N/A

## Type of change

- [X] Set-up

# Testing Instructions

1. `git fetch -a`
1. `git checkout adding-data`
1. ensure data is as needed to begin project
